06/index.html

```diff
@@ -0,0 +1,115 @@
+  <meta name="keywords" content="Digimon, Series, Anime, Adventuras, Monstrous, Niños Elegidos">
+  <title>Tarea con digimons</title>
+  </head>
+  <body>
+    <header>
+      <h1>Digimon</h1>
+      <ul>
+        <li>Intro</li>
+        <li>Favorite Digimons</li>
+        <li>Opening</li>
+        <li>Demo</li>
+      </ul>
+      <p>Digimon, término tomado de Digital Monsters literalmente en español Monstruos Digitales, estilizado como DIGIMON, es una franquicia de medios creado por Akiyoshi Hongo​ que incluye productos como mascotas virtuales, videojuegos, películas, animes, mangas, juguetes y juegos de cartas coleccionables.</p>
+      <p>El argumento de la saga está basado en un mundo paralelo conocido como Mundo Digital, que está habitado por unas criaturas ficticias llamadas Digimon. Son formas de vida artificial creadas a partir de bits y piezas perdidas de datos de ordenadores, que habitan dentro de los aparatos electrónicos que posean memorias no volátiles. Los Digimon son criados por humanos llamados "Tamers" (niños elegidos en Digimon Adventure y en su secuela Digimon Adventure 02) para combatir a los villanos que amenazan con destruir el equilibrio del Mundo Digital.</p>
+    </header>
+    <nav>
+      <h2>Digimon favorite</h2>
+      <ol>
+        <li>Agumon</li>
+        <img src="./assets/Agumon.jpeg">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+        <li>Angemon</li>
+        <img src="./assets/Angemon.webp">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+        <li>Greymon</li>
+        <img src="./assets/Greymon.webp" width="125">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+        <li>Togemon</li>
+        <img src="./assets/Togemon.webp" width="100">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+        <li>Patamon</li>
+        <img src="./assets/Patamon.webp" width="125">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+        <li>Gabumon</li>
+        <img src="./assets/Gabumon.webp" width="100">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+        <li>WereGarurumon</li>
+        <img src="./assets/WereGarurumon.webp" width="100">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+        <li>Angewomon</li>
+        <img src="./assets/Angewomon.webp" width="125">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+        <li>Gomamon</li>
+        <img src="./assets/Gomamon.webp" width="125">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+        <li>Zudomon</li>
+        <img src="./assets/Zudomon.webp" width="150">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+      </ol>
+    </nav>
+    <aside>
+      <h2>Opening</h2>
+      <video src="./assets/Digimon Adventure 01 Opening Español Latino (720p).mp4" controls poster="./assets/digimon miniatura.jpeg"></video>
```

Comment:
This is not the best way of naming a file, remember that almost everything is written in English so the `ñ` does not exist.
Esa no es la mejor forma de nombrar un archivo, recuerda que casi todo está escrito en inglés por lo que la `ñ` no existe.

Remember about casing -> PASCAL, SNAKE, CAMEL, KEBAB